scripts/create_farm.py

```diff
@@ -1,57 +1,71 @@
-import argparse
-
 from get_tree_model import get_tree_model
-
+from itertools import zip_longest
+import matplotlib.pyplot as plt
+from get_plane_model import *
 import numpy as np
+import argparse
+import random
 
 FT_TO_M = 0.3048
 row_width_spacing = 5 * FT_TO_M
 row_length_spacing = 5 * FT_TO_M
 block_to_block_spacing = 15 * FT_TO_M
 
+#### ----- SET THESE VARIABLES ------
```

Comment:
Move this comment above to include the spacing variables too since they are also user set

Reply:
Done